package.json

```diff
@@ -20,6 +20,7 @@
     "next": "^13.0.3",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "swr": "^2.0.0",
```

Comment:
_(arbitrary comment location to start a thread)_

I only see 3 tokens in the dropdown when connected to mainnet, is that expected with this PR?

<img src="https://user-images.githubusercontent.com/17163988/211646172-fc5e698e-bf98-4faf-8ac3-1e4c3550ceba.png" width=400 />

Reply:
Hmm, that actually seems like the changes were missing as those are the placeholder tokens. Mainnet should have a big list and Goerli will only have 2

Reply:
Is this still an issue?

Reply:
Nope, thanks!

<img width="200" alt="image" src="https://user-images.githubusercontent.com/17163988/211972730-e0757108-ec7a-4306-8c2e-dfc0bdf73c6e.png">

<img width="200" alt="image" src="https://user-images.githubusercontent.com/17163988/211972780-43a040b0-95cb-4e2b-81e3-1e74e4994f5f.png">